manta-js/examples/sdk-example/index.ts

```diff
@@ -0,0 +1,346 @@
+import BN from 'bn.js';
+import type { interfaces } from 'manta-extension-sdk';
+import type { Signer as InjectSigner } from '@polkadot/api/types';
+import type { SubmittableExtrinsic } from '@polkadot/api/types';
+
+import {
+  BaseWallet,
+  MantaPayWallet,
+  MantaSbtWallet,
+} from 'manta-extension-sdk';
+
+import {
+  web3Accounts,
+  web3Enable,
+  web3FromSource,
+} from '@polkadot/extension-dapp';
+
+import {
+  get as getIdbData,
+  set as setIdbData,
+  del as delIdbData,
+} from 'idb-keyval';
+
+// const apiEndpoint = 'wss://kwaltz.baikal.testnet.dolphin.training';
+// const nativeTokenDecimals = 18;
```

Comment:
forgotten commented code

Reply:
Because it is an example, not production code, so is this generally deleted?

Reply:
These kinds of comments are more confusing to me than convenient. But it's up to you.

Reply:
ok, I'll remove it